keyboards/handwired/dactyl_manuform/5x6/keymaps/vial/rules.mk

```
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+
```

...disable tap dance.
```suggestion
TAP_DANCE_ENABLE = no

```